server/tests/controller/article/rateArticle.spec.js

```diff
@@ -0,0 +1,105 @@
+import { logInUserDependencies, signUpDependencies } from "@dependencies";
+import chai, { expect } from "chai";
+import models from "@models";
+import app from "@app";
+import { articleSpec, category } from "@fixtures";
+
+const createArticle = async accessToken => {
+  const _category = await models.Categories.create(category);
```

Comment:
why have you adopted this naming convention?

Reply:
There is a global name category in line 5, to differentiate and indicate that _category is only local to this block

Reply:
indicate to who 🤔 . This variable definition has not been used anywhere else so why use it now ?

Reply:
I did not consider a variable starting when underscoring as a convention, from what I read here [using underscore](https://stackoverflow.com/questions/150192/using-underscores-in-java-variables-and-method-names) maybe I should have considered its implication. It has been changed to something more coherent with the codebase